rnote-ui/src/workspacebrowser/filerow/action/duplicate.rs

```diff
@@ -21,6 +21,7 @@ const DUP_REGEX_PATTERN: &str = r"\.dup\d*(?P<rest>(.*))$";
 const DUPLICATE_SUFFIX: &str = ".dup";
 const DOT: &str = ".";
 
+/// Creates a new `duplicate` action
```

Comment:
Isn't this redundant

Reply:
The function name is a little ambiguous when looking at the source file, so this comment aims to clarify that it is the action that is created, not a function that actually duplicate a directory